### PR TITLE
Add Elrond NanoJS binding

### PR DESCRIPTION
**Pull request by pislaProgramming**

### CODE REVIEW

File: packages/hw-app-elrond/README.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+<img src="https://user-images.githubusercontent.com/211411/34776833-6f1ef4da-f618-11e7-8b13-f0697901d6a8.png" height="100" />
+
+[Github](https://github.com/LedgerHQ/ledgerjs/),
+[Ledger Devs Slack](https://ledger-dev.slack.com/)
+
+## @ledgerhq/hw-app-polkadot
+
+Ledger Hardware Wallet Polkadot JavaScript bindings.
```
Comment (haammar-ledger) [inline packages/hw-app-elrond/README.md:8]: Refs to Polkadot, to be updated.

File: packages/hw-app-elrond/README.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+#### Parameters
+
+-   `path` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** a path in BIP 32 format
+-   `message` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** string with an unsigned transaction
+-   `usingHash` **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)** boolean wich indicate if transaction is hash or raw
+
+Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)&lt;[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)>** a signature for transaction
+
+## getAppConfiguration
```
Comment (haammar-ledger) [inline packages/hw-app-elrond/README.md:88]: Everything below seems duplicated, please review.

File: packages/hw-app-elrond/src/Elrond.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+import type Transport from "@ledgerhq/hw-transport";
+import BIPPath from "bip32-path";
+
+const CHUNK_SIZE = 150;
+const CURVE_MASK = 0x80;
+const CLA = 0xed;
+
+const INS = {
+  GET_VERSION: 0x02,
```
Comment (haammar-ledger) [inline packages/hw-app-elrond/src/Elrond.js:11]: Shouldn't `GET_VERSION` be `0x01`, according to https://github.com/ElrondNetwork/ledger-elrond/blob/master/src/main.c#L28? (maybe I get it wrong)

**Review by haammar-ledger — APPROVED** [review]

**Comment by gre** [comment]
👋  Thanks for the contribution.
Please ack that this PR was merged https://github.com/LedgerHQ/ledgerjs/pull/619 and would require to migrate this implementation to TypeScript as well.

Please tell us if you need any help on the matter, the PR linked above includes documentation to apply an auto migration script. @valpinkman or @JunichiSugiura from my team would be happy to help if you reach a blocker.

Thanks

**Review by valpinkman — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -13,7 +13,7 @@
     "test-no-git-diff": "git diff --exit-code || exit 1",
     "doc": "lerna run doc",
     "lint": "eslint packages/**/src --no-error-on-unmatched-pattern --ext .ts,.tsx -c .eslintrc.json",
-    "build": "BABEL_ENV=production && lerna run build",
```
Comment (haammar-ledger) [inline package.json:16]: Do you really need to remove that flag? This change could impact all coins.
Can you maybe apply it in your local `package.json`?

Comment (gre) [reply package.json:16]: that said, i think the initial code is not working at all. 🤔

Comment (pislaProgramming) [reply package.json:16]: Hi! This change was commited by @valpinkman because I had some issues with tests after migration to TypeScript.

Comment (haammar-ledger) [reply package.json:16]: :handshake:

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**